README.md

```diff
@@ -1094,6 +1095,89 @@ Server: spray-can/1.3.3
 }
 ```
 
+## GET /workflows/:version/:id/call-metadata
+
+This endpoint returns a superset of the data from #get-workflowsversionidlogs in essentially the same format.
+In addition to stdout and stderr this also returns inputs, start and end date, backend-specific job id, and
+return code.
+  
+### Notes
+
+- The logs endpoints could possibly be deleted once this is implemented, nobody is calling those anyway (but
+  check with Chet first).
+  
+- It seems strange that callers want inputs to calls but not outputs (other than stdout and stderr).
+
+- We never actually specify the backend on which the call executed.  One might be able to infer that from
```

Comment:
We do implicitly have this information now in that there will be a row in the appropriate %_JOB table for the backend on which the call executed.  Getting that info is probably a bit nontrivial in Slick and as you point out nobody asked for it yet so I'm inclined to skip it for now.  :smile: